modules/dag-l1/src/test/scala/org/tessellation/dag/l1/infrastructure/trust/storage/TrustDbStorageSuite.scala

```diff
@@ -0,0 +1,37 @@
+package org.tessellation.dag.l1.infrastructure.trust.storage
+
+import cats.effect.IO
+
+import org.tessellation.dag.l1.config.types.DBConfig
+import org.tessellation.dag.l1.infrastructure.db.Database
+import org.tessellation.dag.l1.infrastructure.trust.storage.TrustDbStorage
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.TrustDbValues
+import org.tessellation.security.hex.Hex
+
+import ciris.Secret
+import eu.timepit.refined.auto._
+import weaver.SimpleIOSuite
+
+object TrustDbStorageSuite extends SimpleIOSuite {
+  test("Test writing trust values to the database.") {
+    val dbConfig = new DBConfig("org.sqlite.JDBC", "jdbc:sqlite::memory:", "sa", Secret(""))
+    Database.forAsync[IO](dbConfig).use { implicit db =>
+      val trustStorage = TrustDbStorage.make[IO]
+      val peerId = PeerId(Hex("123456789ABCDEF"))
+      val testScore = Option[Double](0.5)
+      val testRating = Option[Double](0.5)
+      val testObservationAdjustment = Option[Double](0.5)
```

Comment:
you can use a generator (arbitrary) for that purpose

Reply:
Added generators for the test.